src/components/Pagination/PaginationList/PaginationList.spec.tsx

```
@@ -0,0 +1,759 @@
+    page: number;
+    count: number;
+    rowsPerPage: number;
+  }) => {
+    const lastPage = Math.ceil(count / rowsPerPage) - 1;
+    return (
+      <div data-testid="mock-pagination">
+        <button
+          data-testid="firstPage"
+          onClick={(e) => onPageChange(e, 0)}
+          disabled={page === 0}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)